src/main/osd/osd_warnings.h

```diff
@@ -20,11 +20,11 @@
 
 #pragma once
 
-#define OSD_WARNINGS_MAX_SIZE 12
+#define OSD_WARNINGS_MAX_SIZE 31
 #define OSD_FORMAT_MESSAGE_BUFFER_SIZE (OSD_WARNINGS_MAX_SIZE + 1)
 
 extern const char CRASHFLIP_WARNING[];
 
-STATIC_ASSERT(OSD_FORMAT_MESSAGE_BUFFER_SIZE <= OSD_ELEMENT_BUFFER_LENGTH, osd_warnings_size_exceeds_buffer_size);
+STATIC_ASSERT(OSD_FORMAT_MESSAGE_BUFFER_SIZE <= OSD_ELEMENT_BUFFER_LENGTH, "osd_warnings_size_exceeds_buffer_size");
```

Comment:
Not necessary to change this. Old static_assert implementation use symbol name instead of string for error (it generated compiler error caused by negative array size)